.stylelintrc.cjs

```diff
@@ -2,7 +2,50 @@
 
 /** @type {import('stylelint').Config} */
 const config = {
-    extends: "@workleap/stylelint-configs"
+    extends: "@workleap/stylelint-configs",
+    overrides: [
+        {
+            files: ["./packages/**/*.css"],
+            rules: {
```

Comment:
why do we need these overrides?

Reply:
https://gsoftdev.atlassian.net/wiki/spaces/TEC/pages/3581640790/LGTM+-+Use+of+CSS+Logical+Properties+in+Workleap+Components

Reply:
It's not part of any shared config since we're the first workleap project doing this.